package.json

```diff
@@ -40,7 +40,8 @@
     ]
   },
   "devDependencies": {
-    "@backstage/cli": "^0.28.0",
+    "@backstage/cli": "^0.29.2",
+    "@changesets/cli": "^2.18.0",
```

Comment:
moved this from normal deps to devDeps, everything should be in devDeps in the root of a monorepo AFAIK.